src/components/TermandPrivacy/PrivacyAndCookies.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { RegulationsCards } from './RegulationsCards';
+
+export const PrivacyAndCookies = () => {
```

Comment:
Why didn't You use an simplified version? () => (

) ?
And where is FC type for this functional component?